### PR TITLE
Added default Input html element component.

### PR DESCRIPTION
**Pull request by Prahitha**

### CODE REVIEW

**Comment by Paarmita** [comment]
@Prahitha Thanks for the PR. This looks good. 
Will you like to work on more input UI in the same PR, or should I merge this one?

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  background-color: ${C.GreyLight};
+  padding-left: 30px;
+  color: ${C.GreySecondaryText};
+  font-weight: ${F.SemiBold};
+  font-size: ${F.XL};
+  border: none;
+  border-radius: 17px;
+
+  ::placeholder,
+  ::-webkit-input-placeholder {
```
Comment (divyanshu-rawat) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input.ts]: Please remove ::-webkit-input-placeholder, ::-ms-input-placeholder,  `::placeholder` is fine ` vendor prefixes` are automatically added in production.

Comment (Prahitha) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input.ts]: Done!

File: src/ignitus-UserInterfaceKit/Components/Atoms/interfaceInputs/Components/interfaceInputs.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { InputContainer } from '../styles';
+import { StyledHeading2 } from '../../../../styles';
+import { InputField1 } from '../../../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input';
+
+const interfaceInputs = () => (
+  <InputContainer>
+  <StyledHeading2>Input Fields ★</StyledHeading2>
+  <hr />
+    <InputField1 placeholder="First Name" type="text" />
```
Comment (divyanshu-rawat) [inline src/ignitus-UserInterfaceKit/Components/Atoms/interfaceInputs/Components/interfaceInputs.tsx]: And please create a component for an input that will receive the `props`  for example have a look here - https://github.com/Ignitus/Ignitus-client/blob/develop/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultCheckbox/Components/defaultCheckbox.tsx

You can't hardcode type="text" so `type` here will be property that can receive a `string` that can string can be `password`, `text` etc, and `placeholder`, `name` will also be a property.

`<input type={type} placeholder={placeholder} name={name}>`


For example see these 2 use cases and how they are being used.

https://github.com/Ignitus/Ignitus-client/blob/develop/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultCheckbox/Components/defaultCheckbox.tsx

https://github.com/Ignitus/Ignitus-client/blob/develop/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultDropdown/Components/defaultDropdown.tsx

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input.ts (hunk 1 above)
Comment (divyanshu-rawat) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/input.ts]: Thanks just left another review :)

**Comment by Paarmita** [comment]
@Prahitha Please rebase your branch with `develop` and solve some relevant hound suggestions as well.

**Comment by Prahitha** [comment]
Thanks for helping out with my first PR.

**Pull request merged**